packages/chaire-lib-frontend/src/services/map/events/GlobalMapEvents.ts

```diff
@@ -6,10 +6,17 @@
  */
 import MapboxGL from 'mapbox-gl';
 import _debounce from 'lodash.debounce';
+import { lineString, bboxPolygon, bbox, feature } from '@turf/turf';
 
 import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
 import Preferences from 'chaire-lib-common/lib/config/Preferences';
 import { MapEventHandlerDescription } from '../IMapEventHandler';
+import { getLinesInView, offsetOverlappingLines } from 'chaire-lib-common/lib/services/geodata/ManageOverlappingLines';
+import { getNodesInView, manageRelocatingNodes } from 'chaire-lib-common/lib/services/geodata/RelocateNodes';
+
+// TODO: Make zoomLimit modifiable by user
+const zoomLimit = 14; //Zoom levels smaller than this will not apply line separation
+let applyAestheticChangesNonce: Object = new Object();
```

Comment:
To avoid the lint error, you can try using the Record<string, unknown> type instead. If it doesn't work a line above saying `// eslint-disable-next-line @typescript-eslint/ban-types` would be acceptable in this case

Reply:
I disabled it, as done in TransitRoutingForm.